sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Internals/Profiling/ProfilingSessionTraceProcessor.cs

```diff
@@ -17,7 +17,11 @@ internal sealed class ProfilingSessionTraceProcessor : BaseProcessor<Activity>
         /// Name of the Tag (and subsequent OpenTelemetry attribute) that
         /// records the profiling session ID.
         /// </summary>
-        internal const string TagName = "profiling.session";
+        /// <remarks>
+        /// Experimental until formalization of OTEP profiles data format.
+        /// See https://github.com/open-telemetry/oteps/pull/239.
```

Comment:
The period at the end makes it difficult to open this link.

```suggestion
        /// See <see href="https://github.com/open-telemetry/oteps/pull/239" />.
```

Reply:
In which IDE was it difficult to open? Originally, I left off the period, but I got a lightbulb (style cop?) saying to add one. So, I added the period and, surprisingly, it opened correctly. In any case, thanks for the tip on "href". I didn't realize it was an option.